Training/Repository/IRepository.cs

```diff
@@ -4,7 +4,8 @@ public interface IRepository<T> where T: class {
     Task<T> Add(T entity);
     Task<T> Update(T entity);
     Task Delete(T entity);
-    Task<T?> GetById(Guid id);
+    Task<T> GetById(Guid id);
```

Comment:
what if it doesnt find an entity?

Reply:
it always find one if not it throws error in the DB

Reply:
refactor it so it uses nulls